config/locales/en.yml

```diff
@@ -37,6 +37,8 @@ en:
   time:
     formats:
       default: '%-d %B %Y'
+      datetime_format: '%Y-%m-%d'
```

Comment:
The format should be without the hyphens

Reply:
Are you sure? The sample html from Peter was: "<time datetime='2017-06-13'>13 June 2017</time>".

Reply:
Sorry, you're right - that was my mis-reading a while ago